rsconnect/api.py

```diff
@@ -536,17 +576,19 @@ def __init__(
         secret: Optional[str] = None,
         timeout: int = 30,
         logger: Optional[logging.Logger] = console_logger,
-        **kwargs: Any,
+        extra_kwargs: ExecutorKwargs | None = None,
```

Comment:
Can we add the rest of the arguments as optionals?

Reply:
Sure thing.